Content.Server/GhostTypes/GhostSpriteStateComponent.cs

```diff
@@ -0,0 +1,14 @@
+namespace Content.Server.GhostTypes;
+
+/// <summary>
+/// Changes the entity sprite according to damage taken
+/// </summary>
+[RegisterComponent]
+public sealed partial class GhostSpriteStateComponent : Component
+{
+    /// <summary>
+    /// Prefix the system will add to the damage name it's using
```

Comment:
This should be more specific. What is "the damage name"? What should someone put here?